src/contracts/pack.ts

```diff
@@ -131,7 +131,7 @@ export class Pack implements UpdateableNetwork {
    * @param packId - the id of the pack to fetch
    * @returns the pack metadata
    */
-  public async get(packId: string): Promise<PackMetadata> {
+  public async get(packId: BigNumberish): Promise<PackMetadata> {
```

Comment:
let's make this change separately no? seems like a bugfix we should ship independently

Reply:
https://github.com/thirdweb-dev/typescript-sdk/pull/317